test/EndToEndTests/Tests/Client/Build.Desktop/ODataWCFServiceTests/IDReadLinkEditLinkTests.cs

```diff
@@ -39,9 +39,9 @@ public void ClientShouldUseTheEditLinkGotFromPayloadToUpdateTheEntryInMinimalFul
         private void UpdateObject(string mimeType)
         {
             TestClientContext.MergeOption = MergeOption.OverwriteChanges;
-            TestClientContext.SendingRequest2 += (sender, eventArgs) => ((Microsoft.OData.Client.HttpWebRequestMessage)eventArgs.RequestMessage).SetHeader(TestHeader, "EditLink");
+            TestClientContext.SendingRequest2 += (sender, eventArgs) => ((Microsoft.OData.Client.HttpClientRequestMessage)eventArgs.RequestMessage).SetHeader(TestHeader, "EditLink");

```

Comment:
Any reason we couldn't import the `Microsoft.OData.Client` namespace instead of typing the whole namespace so many places?

Reply:
Hmm. I don't know why this was done, I was just focused on changing the referenced class. I guess I could import the namespace as well.